drivers/gles2/shaders/scene.glsl

```diff
@@ -2467,6 +2467,12 @@ FRAGMENT_SHADER_CODE
 	frag_color.rgb = mix(pow((frag_color.rgb + vec3(0.055)) * (1.0 / (1.0 + 0.055)), vec3(2.4)), frag_color.rgb * (1.0 / 12.92), vec3(lessThan(frag_color.rgb, vec3(0.04045))));
 #endif
 
+	// Write to the final output once and only once.
+	// Use a temporary in the rest of the shader.
+	// This is for drivers that have a performance drop
+	// when the output is read during the shader.
+	gl_FragColor = frag_color;
```

Comment:
Moved this because it turns out `RENDER_DEPTH` is actually depth only pass, and we don't want to write to color in depth only pass.